src/azul/terraform.py

```diff
@@ -793,6 +793,17 @@ def tf_config(self, app_name):
         del deployment['lifecycle']
         deployment['triggers'] = {'redeployment': deployment.pop('stage_description')}
 
+        # Using these AWS API Gateway extensions to the OpenAPI specification
+        # avoids maintaining a complicated `replace_triggered_by` and `triggers`
+        # dependency between Terraform resources in order to ensure that the API
+        # gateway is deployed when these properties are modified.
```

Comment:
```suggestion
        # Using these AWS API Gateway extensions to the OpenAPI specification
        # eliminates the need to maintain a complicated `replace_triggered_by` and
        # `triggers` dependency between Terraform resources, thereby ensuring the
        # API Gateway resource is deployed when these properties are modified.
```

or as you may see appropriate.

Reply:
Since there is only one of these API Gateway extensions being added in this PR, I changed the verbiage to be singular. Later, when other extensions are added in we can change the comment to be plural (e.g. "this property" -> "these properties").